src/components/Container/Container.css

```diff
@@ -0,0 +1,7 @@
+.movie-grid {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: 3%;
+  margin: auto;
+  width: 90%;
+}
```

Comment:
Let's revisit this soon because the dimensions of the grid don't currently play nicely with the dimensions of the movie posters. Lots of unused space.